### PR TITLE
[test_gap][bgp] Add testcase to verify dynamic neighbor restoration

### LINKED ISSUE #3586: Need new test: Fast-Reload
Dynamic BGP sessions will be restored after fast/warm reboot.

### PR DESCRIPTION
**Pull request by lolyu**
### Description of PR

Summary:
Fixes #3586

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
Verify dynamic BGP session is restored after fast/warm reboot

Signed-off-by: Longxiang Lyu <lolv@microsoft.com>

### CODE REVIEW

**Comment by lguohan** [comment]
can you add this to kvm testbed?

**Comment by lolyu** [comment]
> can you add this to kvm testbed?

Sure, will update.

File: tests/bgp/test_bgp_slb.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+from tests.common import reboot
+from tests.common.helpers.bgp import BGPNeighbor
+from tests.common.dualtor.mux_simulator_control import mux_server_url                                   # lgtm[py/unused-import]
+from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports_to_rand_selected_tor  # lgtm[py/unused-import]
+from tests.common.utilities import wait_until
+
+
+pytestmark = [
+    pytest.mark.topology("t0")
+]
```
Comment (wangxin) [inline tests/bgp/test_bgp_slb.py:12]: If it is supported on VS setup, suggest to add below mark:

```
pytest.mark.device_type('vs')
```

Comment (lolyu) [reply tests/bgp/test_bgp_slb.py:13]: updated, thanks~

**Review by wangxin — APPROVED** [review]

**Pull request merged**